### ISSUE TITLE
What is a miRNA?

### ISSUE THREAD

**Issue by ThomasDesvignes** [miRTop/incubator#9]
"What is a miRNA?" is a big open question! When we think we've nailed it down, new evidence shows us new exceptions or variations. How should we deal with that? What should we do?

Everyone has different vision because different interests in studying miRNAs. Some see miRNAs as evolutionary genetic entities and care a lot about their pure genetic nature and biogenesis, some see them as functional products and don't care that much about how they were made but instead care more about what they do, some others can see them in a totally different way, or as a mix of all other visions. 

Comments, suggestions, personal visions and interests, please share it here!

**Comment by lpantano** [comment]
Hi all again, 

cc: @ThomasDesvignes @mhalushka @mlhack @keilbeck @BastianFromm @ivlachos @TJU-CMC

The first question that came was this. Thomas already created an issue, so I am going to re-use. 
I will define a little more the range for answering, re-phrasing the question:

* "Do we need to define what is a miRNA before jump into naming?"

**Deadline to participate: May,01.** 

* Suggested answer: a) we should do it before moving forward, b) we can move forward. Please, add your thoughts about what you think will help to define real miRNAs.

My position here is that this is mainly for the group that will/do maintain the miRNA database. Right now, the group doesn't have the resources to be involved on that and I think we can jump into find a better naming for miRNA even if we don't have a total 100% accurate DB, although miRGeneDB is working toward that direction.

I think this could be part of the naming, miRNAs that have been considered like miRNA but they are not, and the ones that are well described. At the end, only few experiments will support or not the real existence of the miRNA and its function, and probably the DB that ends up supporting miRNAs would need to decide (like miRBase is/was doing). 

Said that, (I think) the community we are creating can help to create collaboration between researchers who want to work more deeply on this, and define real miRNA from possible-artifact miRNA. The group is a good place to discuss further and help to deploy the information around the different groups. That will ensure a better dispersion of the information and make it usable enough, since the input of all of us should be enough to address the main concerns nowadays.

My conclusion is that we can move forward to discuss miRNA naming, and considering adding this information somewhat in the name or database because it's relevant. 

My two cents for the main question: I think miRNA should be differentiated from miRNA-like sequences that can be loaded into RISC complexes even if they are tRNA or sncRNA. Working with evolution and secondary structure that we already know from them, should help to define miRNA from other novel small RNA with same regulatory functions. Naming other small RNAs that act as miRNAs can be a downstream project if we end up finding a solution for miRNA (Again, maybe some people around is interesting to lead this project at some point). 

**Deadline to participate: May,01.**

**Comment by ThomasDesvignes** [comment]
Hi all,

My position on this issue follows Lorena's: I think we can, and should, move forward and we don't need to solve the definition of the nature of miRNAs to be able to do things on isomiRs. 

The system we will come up with to differentiate and describe isomiRs will be in respect to the miRNA they differ from. Whether the said miRNA is or isn't validated as a canonical miRNA shouldn't impair our system otherwise it means it's not a system robust enough to new development. Robustness is to me a key point because, for sure, we will continue to see myriads of new putative miRNAs added to the list of known miRNAs but missing experimental data validating that they are both Drosha and Dicer dependent and incorporated into the RISC, and some will later be confirmed to be canonical miRNAs and some won't. Being able to remain a robust system even if the nature of the miRNA-like fragment changes is crucial. Therefore, I think we don't have to agree on the definition of a miRNA to be able to move forward our isomiR interests.

**Comment by mhalushka** [comment]
I agree with the above.  A robust isomiR naming convention can stand apart from whatever decisions are made to revise the rules for what is/is not a miRNA.  Does anyone have a running list of published isomiR naming methods?  I know the Rigoutsos lab has one, but I believe I have seen others.  It would be useful to know the benefits and limitations of what has already been established.

**Comment by ivlachos** [comment]
The only limitation is the reference sequence, since an isomiR is usually named vs a mature miRNA sequence that is considered as the standard mature product from that hairpin arm. If this consensus sequence changes (as more experiments are taken into account or criteria change) then isomiR naming will have also to change accordingly. I restrain myself from elaborating more here, since I believe that this is a  separate topic (which should be the reference mature sequences for a pre-miRNA) but it is also highly related (will isomiRs be named based on a reference mature miRNA sequence - which one should be selected - what happens if it changes).

**Comment by mlhack** [comment]
This is from Bastian Fromm and me,

I agree as well that we could go on with the naming of microRNAs and its isomiRs if we cannot agree on how to assess the question on what a real microRNA is right now. 

But maybe we could add a ‘confidence tag’ in the annotation. this could be something like 

i) NGS + structure (‘looks like’ a microRNA due to 5’ homogeneity and Drohsa/Dicer pattern), 
ii) phylogenetic footprinting (miRNA is conserved), 
iii) Ago loading 
iv) Impact of Drosha/Dicer knockout and 
v) positive functional assay.

So the least requirement for a microRNA would be the detection by means of NGS. In this way, only microRNAs that accomplish with this minimum requirement would be in a database. 

Regarding isomiRs, only those that are independently regulated and having a different function in comparison to the canonical sequence should be annotated. For all others, a description (like in sRNAbench) should be sufficient. 

On another note, we have to point out that although we can agree to move forward developing a system for isoform annotation, it will be necessary to have a system and rules in place that annotates novel miRNAs. 
...before one is able to name the corresponding isoforms

**Comment by lpantano** [comment]
Thanks all for the comments!

I will create the next question tomorrow. There is a little overlap here, what is good, because we can move easily to the next one. 

I note that we need a way to integrate the novel/known miRNA somehow, and we can discuss this in the proper issue starting with the ideas all you gave here.

Cheers
